src/katex.less

```diff
@@ -411,6 +404,7 @@
             display: inline-block;
             margin: 0 -0.025em;
             border-right: 0.05em solid;
+            min-width: 1px;
```

Comment:
Now that this line is far away from the rest, maybe it needs a comment too?

Reply:
Done.